### DISCUSSION TITLE
Episode 8: A Talk with Tomasz Godzik

### DISCUSSION THREAD

**Discussion by ckipp01** [ckipp01/tooling-talks#16] [Announcements]
![A talk with Tomasz (twitter)](https://user-images.githubusercontent.com/13974112/148215158-c9843240-81e1-4aeb-ae4a-7051f0965a80.jpg)

It's a new year with some new guests! Happy to have Tomasz be the first guest of the year! Drop any questions you might want asked here!

### Find Tomasz Online

- [@TomekGodzik](https://twitter.com/TomekGodzik) on Twitter
- @tgodzik on GitHub

**Comment by mtk** [comment]
Where is the best place to get a beer after work in Krakow?  :-)

**Reply by tgodzik** [reply]
Plenty of good places :D One of my favourite is Muhabet, a mix of a turkish mezze with some nice craft beer.

**Comment by keynmol** [comment]
Here's my question (for the podcast):

How can we motivate more people to work on developer tooling? What can make it easier, more attractive, and less daunting?

**Reply by ckipp01** [reply]
Haha, you were literally hours too late. We recorded last night. _However_ we did touch on some of this!
